app/src/main/python/gogdl/auth.py

```
@@ -0,0 +1,133 @@
+        credentials = self.credentials_data.get(CLIENT_ID)
+        if not credentials or "refresh_token" not in credentials:
+            self.logger.error("No refresh token available")
+            return False
+
+        refresh_token = credentials["refresh_token"]
+        url = f"https://auth.gog.com/token?client_id={client_id}&client_secret={client_secret}&grant_type=refresh_token&refresh_token={refresh_token}"
+
+        try:
+            response = requests.get(url, timeout=10)
+        except (requests.ConnectionError, requests.Timeout):
+            self.logger.error("Failed to refresh credentials")
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)